[2024-09-02T07:30:39.607+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-02T07:30:39.627+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_cathcup_backfill_v1.task1 manual__2024-09-02T07:30:38.885915+00:00 [queued]>
[2024-09-02T07:30:39.637+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_cathcup_backfill_v1.task1 manual__2024-09-02T07:30:38.885915+00:00 [queued]>
[2024-09-02T07:30:39.637+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 6
[2024-09-02T07:30:39.656+0000] {taskinstance.py:2879} INFO - Executing <Task(BashOperator): task1> on 2024-09-02 07:30:38.885915+00:00
[2024-09-02T07:30:39.666+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=3385) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-02T07:30:39.665+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_cathcup_backfill_v1', 'task1', 'manual__2024-09-02T07:30:38.885915+00:00', '--job-id', '431', '--raw', '--subdir', 'DAGS_FOLDER/cathcup_and_backfill_dag.py', '--cfg-path', '/tmp/tmp836mbes_']
[2024-09-02T07:30:39.667+0000] {standard_task_runner.py:72} INFO - Started process 3390 to run task
[2024-09-02T07:30:39.668+0000] {standard_task_runner.py:105} INFO - Job 431: Subtask task1
[2024-09-02T07:30:39.712+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_cathcup_backfill_v1.task1 manual__2024-09-02T07:30:38.885915+00:00 [running]> on host efc48c282cbe
[2024-09-02T07:30:39.792+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ubeyd' AIRFLOW_CTX_DAG_ID='dag_with_cathcup_backfill_v1' AIRFLOW_CTX_TASK_ID='task1' AIRFLOW_CTX_EXECUTION_DATE='2024-09-02T07:30:38.885915+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-02T07:30:38.885915+00:00'
[2024-09-02T07:30:39.793+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-02T07:30:39.806+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-02T07:30:39.807+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo This is a simple bash command!']
[2024-09-02T07:30:39.817+0000] {subprocess.py:86} INFO - Output:
[2024-09-02T07:30:39.819+0000] {subprocess.py:93} INFO - This is a simple bash command!
[2024-09-02T07:30:39.819+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-09-02T07:30:39.843+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-02T07:30:39.844+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_with_cathcup_backfill_v1, task_id=task1, run_id=manual__2024-09-02T07:30:38.885915+00:00, execution_date=20240902T073038, start_date=20240902T073039, end_date=20240902T073039
[2024-09-02T07:30:39.882+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-09-02T07:30:39.897+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-02T07:30:39.901+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
